docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -309,6 +318,20 @@ repository as well as each subsequent push to each pull request.
 To pass information from the upstream patch to the downstream patch use
 [downstream_expansions.set](Project-Commands.md#downstream_expansionsset)
 
+**Example**:  to allow testing Spruce tasks as part of patches for the Evergreen project,  
+a configuration like this could be added to **Evergreen's project page:**
+
+![patch_trigger_alias_example.png](../images/patch-trigger-alias-example.png)
+
+This makes it possible to optionally add tasks matching the defined regex to any patch, and because 
+"Add to GitHub Trigger Alias" is checked, meaning that these tasks will be included as part of PR patches. You can also see what Project Trigger Aliases are configured to run on PRs
+
+and because "evergreen" is defined as a module in the yaml (pictured below), the Spruce tasks will incorporate the patch changes.  
+
+![module_example.png](../images/module-example.png)
```

Comment:
Let's show the new format of specifying owner and repo, like shown here
https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Commands#gitget_project

Reply:
done